source/tutorial/triggers-fts.txt

```diff
@@ -1,8 +1,8 @@
 .. _triggers-fts-tutorial:
 
-====================================================
-Build an Application Using Triggers and Atlas Search
-====================================================
+==========================================
+Build Reverse Search Into Your Application
```

Comment:
Non-blocking: Do we want to keep Triggers and Atlas Search in the title? "Build Reverse Search Into Your Application Using Triggers and Atlas Search"

Reply:
The title would spill into two lines if we keep "Using Triggers and Atlas Search." I figured since we mention triggers and Atlas Search in the first sentence, it'd be ok to omit it from the title similar to some of the other tutorials like [ this one](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-30027/tutorial/backend/).

Reply:
sounds good to me